rdr_service/model/bq_site.py

```diff
@@ -24,6 +24,20 @@ class BQDigitalSchedulingStatus(Enum):
     ACTIVE = 1
     INACTIVE = 2
 
+class BQInPersonOperationsStatus(Enum):
```

Comment:
Note: The reason we need to "duplicate" enum class definitions in the PDR generator is because it uses the Python3 `Enum` class vs. the older RDR `messages.Enum` class, and there are differences in the object behavior.  We do this for convenience.